packages/gatsby-plugin-image/src/image-utils.ts

```diff
@@ -194,11 +194,33 @@ export function setDefaultDimensions(
 
     if (aspectRatio && !height) {
       height = Math.round(width / aspectRatio)
+    } else if (!aspectRatio) {
+      aspectRatio = width / height
     }
   }
   return { ...args, width, height, aspectRatio, layout, formats }
 }
 
+/**
+ * Use this for getting an image for the blurred placeholder. This ensures the
+ * aspect ratio and crop match the main image
+ */
+export function getLowResolutionImageURL(
+  args: IGatsbyImageHelperArgs,
+  width = 20
```

Comment:
Any reason not to add this to the setDefaultDimensions call?

Reply:
We don't normally want to default to 20px images, just in this low res helper

Reply:
Ah ok, realize that's reused now.